src/PlaceProperties.ts

```diff
@@ -70,6 +72,10 @@ export interface PlaceProperties {
    * URL of the original data source’s website describing this place.
    */
   infoPageUrl?: LocalizedString;
+  /**
+   * URL of the original data source’s website describing this place.
+   */
+  hasInfoPageUrl?: Boolean;
```

Comment:
I think the attribute name could be misunderstood… what did we want to express here originally?